installer/helm/chart/volcano/config/volcano-scheduler-ci.conf

```diff
@@ -1,4 +1,11 @@
-actions: "enqueue, reclaim, allocate, backfill, preempt"
+actions:
```

Comment:
add version to configuration file, and different version of conf will use differen parser for it.

Reply:
@k82cn , Need to add version in the latest config file only.we should not add version into the old conf file because backword compatability doesn't work.

 any how currently we are just using yaml.Unmarshal() for parsing the config. currently the coding as  follows. first Unmarshal for latest conf if fails try for  old conf file format.


if err := yaml.Unmarshal(buf, &schedulerConfv2); err != nil {
		if err := yaml.Unmarshal(buf, &schedulerConf); err != nil {
			return nil, err
		}
		config.Version = framework.SchedulerConfigVersion1
		config.V1Conf = &schedulerConf
		Tiers = schedulerConf.Tiers
	} else {
		config.Version = framework.SchedulerConfigVersion2
		config.V2Conf = &schedulerConfv2
		Tiers = schedulerConfv2.Tiers
	}

is this okay or something else you are expecting?

Reply:
The best practice is like what kubernetes currently does: maintain a internal unversioned API,  and multiple  versioned apis, the versioned one is used by user. The convertion code is mostly generated by tools. 

FYI:  
```
// +k8s:conversion-gen=k8s.io/kubernetes/pkg/controller/volume/persistentvolume/config
// +k8s:conversion-gen-external-types=k8s.io/kube-controller-manager/config/v1alpha1
```

Reply:
>  Need to add version in the latest config file only.we should not add version into the old conf file because backword compatability doesn't work.

If no version, the default version is v1.

Reply:
@k82cn , do i need to follow the k8s way as mentioned by hzxuzhonghu  or just present code is enough after adding the version into conf file as follows

version: v2
actions:
- name: enqueue
  arguments:
    enqueue-action-idleres-mul: 1.2
- name: allocate
- name: backfill
tiers:
- plugins:
  - name: priority
  - name: gang
  - name: conformance
- plugins:
  - name: drf
  - name: predicates
  - name: proportion
  - name: nodeorder

Reply:
your format is ok to me; k8s's way is too complex, we can do it later.